docs/index.md

```diff
@@ -68,6 +68,68 @@ connection "heroku" {
 - `email` - Email address of the Heroku user.
 - `api_key` - API key (or password) of the Heroku user.
 
+## Multi-Account Connections
+
+You may create multiple heroku connections:
+
+```hcl
+connection "heroku_dev" {
+  plugin  = "heroku"
+  email   = "ryan@dundermifflin.com"
+  api_key = "34c12972-9d18-654c-3ae5-5293ae1889be"
```

Comment:
@madhushreeray30 could you please ad different API keys to each connection, so that they appear like different connections?

Reply:
> @madhushreeray30 could you please ad different API keys to each connection, so that they appear like different connections?

@misraved I have changed partially.